@shared/api/internal.ts

```diff
@@ -220,21 +220,24 @@ export const getAccountBalances = async ({
 
   let balances = null;
   let isFunded = null;
+  let subentryCount = 0;
 
   try {
-    ({ balances } = await dataProvider.fetchAccountDetails());
+    ({ balances, subentryCount } = await dataProvider.fetchAccountDetails());
```

Comment:
added saving the subEntryCount instead of creating another method calling accountDetails. I think `getAccountBalances` still applies here